lib/afip_bill/generator.rb

```diff
@@ -13,6 +13,7 @@ class Generator
     FOOTER_PATH = File.dirname(__FILE__) + '/views/shared/_factura_footer.html.erb'.freeze
     BRAVO_CBTE_TIPO = { "01" => "Factura A", "06" => "Factura B" }.freeze
     IVA = 21.freeze
+    PRODUCT_CONCEPT_CODE = '01'.freeze
```

Comment:
@tcordeu Is this specific to your application or to the AFIP concepts? I don't follow the reasoning behind this. Could you please point me in the right direction? Trying to understand if this belongs in the `afip_bill` gem or in your application code.

Reply:
@etagwerker I'm not sure if AFIP uses that code (`01`) but that's what the `bravo` gem uses when generates the json for the bill: https://github.com/vurbia/bravo/blob/a7db5d7109d24cc0b0a7ad3e50a764f2f96d88ff/lib/bravo/constants.rb#L30

We should probably have a constant similar to that one if we want support for the concept.

Also we should update the README file since we currently don't mention the `concepto` key in https://github.com/ombulabs/afip_bill/blob/master/README.md#setup